src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs

```diff
@@ -245,8 +245,7 @@ public SyndicationTextInput TextInput
         private SyndicationLink TryReadDocumentationFromExtension(SyndicationElementExtensionCollection elementExtensions)
         {
             SyndicationElementExtension documentationElement = elementExtensions
-                                      .Where(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace)
-                                      .FirstOrDefault();
+                                      .FirstOrDefault(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace);
```

Comment:
This isn't dead code, it's a different way of writing the original.

Are there tests validating this?

Are they equivalent performance-wise?  i.e. does FirstOrDefault have all of Where's optimizations that would apply in this situation?

Etc.

Reply:
- These all have 100% test coverage in https://github.com/dotnet/corefx/pull/35107

Here's the code https://github.com/dotnet/corefx/blob/master/src/System.Linq/src/System/Linq/First.cs#L75-L103 that gets the first.

We'd previously allocate an iterator here: https://github.com/dotnet/corefx/blob/master/src/System.Linq/src/System/Linq/Where.cs#L42

So actually we're saving allocations here